mirage/factories/registration.ts

```diff
@@ -154,6 +154,10 @@ export default NodeFactory.extend<MirageRegistration & RegistrationTraits>({
         return faker.internet.url();
     },
 
+    providerSpecificMetadata() {
+        return [];
```

Comment:
Since we're not showing how this is used here, could you add some metadata to one of the providers (maybe EGAP) in the mirage default file?

Reply:
Do you mean adding it to one of the registrations?

Reply:
Oh, right. Yes, sorry.

Reply:
`default.ts` is updated now.